moveit_simple/include/moveit_simple/robot.h

```diff
@@ -538,6 +540,7 @@ class Robot
     const std::unique_ptr<JointTrajectoryPoint> &to, double t,
     std::unique_ptr<JointTrajectoryPoint> &point) const;
 
+  // TODO(mlautman): deprecate this
```

Comment:
Why should this be deprecated? I thought this would just be passed to the TrajectoryPlanner